app/src/main/java/org/luxoft/sdl_core/CommunicationService.java

```diff
@@ -68,85 +83,95 @@ public void onReceive(Context context, Intent intent) {
                 }
 
                 switch (intent.getAction()) {
+
                     case ACTION_START_BLE:
                         Log.i(TAG, "ACTION_START_BLE received by communicationServiceReceiver");
-                        mNativeBleAdapterThread = new JavaToNativeBleAdapter(CommunicationService.this);
-                        mNativeBleAdapterThread.start();
+                        mCurrentTransport = TransportType.BLE;
+                        mNativeAdapterThread = new JavaToNativeAdapter(CommunicationService.this,
+                                BLE_SENDER_SOCKET_ADDRESS, BLE_RECEIVER_SOCKET_ADDRESS);
+                        mNativeAdapterThread.start();
+                        break;
+
+                    case ACTION_START_BT:
+                        Log.i(TAG, "ACTION_START_BT received by communicationServiceReceiver");
+                        mCurrentTransport = TransportType.CLASSIC_BT;
+                        mNativeAdapterThread = new JavaToNativeAdapter(CommunicationService.this,
+                                BLE_SENDER_SOCKET_ADDRESS, BLE_RECEIVER_SOCKET_ADDRESS);
```

Comment:
@ychernysheva as you already have these variables:
```suggestion
                                BT_SENDER_SOCKET_ADDRESS, BT_RECEIVER_SOCKET_ADDRESS);
```